### DISCUSSION TITLE
I need some help for NequIP training with equation of state NPT AIMD

### DISCUSSION THREAD

**Discussion by turbosonics** [mir-group/nequip#395] [Q&A]
Hello, 

I'm testing various training set configurations for NequIP training. 

When I mix equation of state NPT AIMD data (-20% ~ 20% volume compression and expansion of unit cell, 10000 frames) with NVT AIMD data, I found that all losses and errors from NequIP training drop to 0 around 10~30 epochs, and this happens regardless of several different hyperparameter conditions and loss coefficients I tested. I'm suspicious this might be overfitting, but not 100% sure. When I don't mix them and run separately, both trainings work fine, errors and loss smoothly converges. This happens only when I attempt to train together. 

I think I'm wrong with hyperparameter setting, but not sure which setting would help me to prevent the current possible overfitting problem. If anyone have some experience of NequIP training with equation of state NPT AIMD, any suggestions?
